home.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The LOTR Lovers Conference</title>
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <div>
+        <div class="popUp flex">
+            <ul id="popUpItems" class="popUpItems flex">
+               <a href="#about">
+                 <li class="popUpItem">
+                About
+                </li>
+            </a>
+            <a href="#program">
+                <li class="popUpItem">
+               Program
+               </li>
+           </a>
+           <a href="#join">
+            <li class="popUpItem">
+           Join
+           </li>
+       </a>
+       <a href="#sponsors">
+        <li class="popUpItem">
+       Sponsor
+       </li>
+    </a>
+    <a href="#news">
+        <li class="popUpItem">
+       News
+       </li>
+    </a>
+            </ul>
+            <p class="popUpCancel">X</p>
+        </div>
+    </div>
+    <div class="hamburger">
+        <img src="icons8-hamburger-32.png" class="hamburgericon" id="hamburgericon" alt="hamburgericon"> 
+    </div>
+    <div class="navigationBar">
+        <img class="navigationLogo" src="mellonLogo.png">
+        <ul class="navigationItems flex">
+           <a href="#about"> <li class="navigationItem">
+                About
+            </li></a>
+           <a href="#program"> <li class="navigationItem">
+                Program
+            </li></a>
+            <a href="#join"> <li class="navigationItem">
+                Join
+            </li></a>
+           <a href="#sponsor">
+           <a href="#sponsor"> <li class="navigationItem">
+                Sponsor
+            </li>
+        </a>
+        </a>
+           <a href="#news"> <li class="navigationItem">
+                News
+            </li>
+        </a>
+        </ul>
+    </div>
+<header class="header flex">
+    <h1 class="mainTitle">
+        "Mellon!"
+    </h1>
+    <h2 class="secondTitle">
+        ARE YOU READY  A JOURNEY TO  MIDDLE-EARTH?
+    </h2>
+</header>
+<main>
+    <div class="index flex">
+        <p class="description">
+            As a Tolkien reader or  a Lord of the Rings watcher, we invite you a fantastic journey in Istanbul with different people all over the world.   
+        </p>
+        <p class="date">
+            2024.03.01 (WED) - 05 (FRI)</p>
+        <p class="location">
+@Taksim Square of İstanbul in Turkey
+        </p>
+    </div>
```

Comment:
- [ ] Kindly add some margin/ padding top and bottom to your section so that it matches the provided Behance design. The sections almost overlap with the ones above or bottom.

**This is how your section looked like:**

![image](https://user-images.githubusercontent.com/86318284/212124383-d8176113-1c65-4e76-9d67-60619da71607.png)


**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/212124457-8b396af2-37b3-42e3-bcd8-bc61e4875548.png)